stdlib/public/Darwin/Foundation/Foundation.swift

```diff
@@ -29,7 +29,153 @@ public let NSNotFound: Int = .max
+///   instead when `value` is the empty string.
+///
+/// Export Localizations with Xcode
+/// -------------------------------
+///
+/// Xcode can read through a project's code to find invocations of
+/// `NSLocalizedString(_:tableName:bundle:value:comment:)` and automatically
+/// generate the appropriate strings files for the project's base localization.
+///
+/// In Xcode, open the project file and, in the `Edit` menu, select
+/// `Export for Localization`. This will generate an XLIFF bundle containing
+/// strings files derived from your code along with other localizable assets.
+/// `xcodebuild` can also be used to generate the localization bundle from the
+/// command line with the `exportLocalizations` option.
+///
+///     xcodebuild -exportLocalizations -project <projectname>.xcodeproj \
+///                                     -localizationPath <path>
+///
+/// These bundles can be sent to translators for localization, and then
+/// reimported into your Xcode project. In Xcode, open the project file. In the
+/// `Edit` menu, select `Import Localizations...`, and select the XLIFF
+/// folder to import. You can also use `xcodebuild` to import localizations with
+/// the `importLocalizations` option.
+///
+///     xcodebuild -importLocalizations -project <projectname>.xcodeproj \
+///                                     -localizationPath <path>
+///
+/// Choose Meaningful Keys
+/// ----------------------
+///
+/// Words can often have multiple different meanings depending on the context
+/// in which they're used. For example, the word "Book" can be used as a noun—a
+/// printed literary work—and it can be used as a verb—the action of making a
+/// reservation. Words with different meanings which share the same spelling are
+/// heteronyms.
+///
+/// Different languages often have different heteronyms. "Book" in English is
+/// one such heteronym, but that's not so in French, where the noun translates
+/// to "Livre", and the verb translates to "Réserver". For this reason, it's
+/// important make sure that each use of the same phrase is translated
+/// appropriately for its context by assigning unique keys to each phrase and
+/// adding a description comment describing how that phrase is used.
+///
+///     NSLocalizedString("book-tag-title", value: "Book", comment: """
+///     noun: A label attached to literary items in the library.
+///     """)
+///
+///     NSLocalizedString("book-button-title", value: "Book", comment: """
+///     verb: Title of the button that makes a reservation.
+///     """)
+///
+/// Use Only String Literals
+/// ------------------------
+///
+/// String literal values must be used with `key`, `tableName`, `value`, and
+/// `comment`.
+///
+/// Xcode does not evaluate interpolated strings and string variables when
+/// generating strings files from code. Using those language features will
+/// result in exported localizations that resemble the code expression instead
```

Comment:
There's a verb tense disagreement here between "localizations" and "its value". Maybe this instead?

```suggestion
/// result in an exported localization that resembles the code expression instead
```